app/components/special-coverage/special-coverage-edit/special-coverage-edit.html

```diff
@@ -107,9 +107,8 @@
         <label for="specialCoverageCampaign">Sponsor Campaign</label>
         <autocomplete-basic
             class="form-control"
-            ng-if="!model.campaign || model.campaign.$resolved"
```

Comment:
Per the thing above about it flashing, maybe try leaving `ng-if` in as `ng-if="!model.tunicCampaignId"`